config/jobs/kubernetes/sig-testing/kind.yaml

```diff
@@ -6,7 +6,7 @@ periodics:
     preset-service-account: "true"
     preset-bazel-scratch-dir: "true"
     preset-bazel-remote-cache-enabled: "true"
-    preset-dind-memory-backed: "true"
+    preset-dind-enabled: "true"
```

Comment:
this didn't really help AFAICT, and eats more memory. we should probably remove the preset in a follow-up